src/redprl/redprl.grm

```diff
@@ -456,12 +479,8 @@ atomicRawTac
   | RULE_LEMMA LBRACKET OPNAME customOpParams RBRACKET (Ast.$$ (O.POLY (O.RULE_LEMMA (OPNAME, customOpParams)), []))
   | RULE_LEMMA OPNAME (Ast.$$ (O.POLY (O.RULE_LEMMA (OPNAME, [])), []))
 
-  | LPAREN LBRACKET OPNAME customOpParams RBRACKET bindings RPAREN (Ast.$$ (O.POLY (O.CUST (OPNAME, customOpParams, NONE)), bindings))
-  | LPAREN OPNAME bindings RPAREN (Ast.$$ (O.POLY (O.CUST (OPNAME, [], NONE)), bindings))
-
-  | operator (Ast.$$ (operator, []))
```

Comment:
(what I mean is, a *tactic* expression like `if x then else` generates two subgoals, but this is because we parse the empty space as the identity tactic, and you can fill it in with a real tactic as you get further along.)